.yarnrc.yml

```diff
@@ -1 +1,3 @@
-nodeLinker: node-modules
\ No newline at end of file
+nodeLinker: node-modules
+
+yarnPath: .yarn/releases/yarn-3.5.1.cjs
```

Comment:
We should not be adding the yarn executable to our repository. This approach seems incorrect.

Reply:
Lol